openslides/core/static/js/core/base.js

```diff
@@ -48,10 +48,12 @@ angular.module('OpenSlidesApp.core', [
     'REALM',
     'ProjectorID',
     '$q',
+    '$timeout',
+    'operator',
     'ErrorMessage',
-    function (DS, REALM, ProjectorID, $q, ErrorMessage) {
+    function (DS, REALM, ProjectorID, $q, $timeout, operator, ErrorMessage) {
         var socket = null;
-        var recInterval = null;
+        var retryConnectCallbacks = [];
```

Comment:
But now we have the option ;). I need to hook into this from the `OpenSlides` factory, because I cannot use this factory as a dependency in `autoupdate`. This leads to circular dependencies.
And if I make the option to register one ore more callbacks - this is not the point I think.